optimum/habana/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .pipeline_stable_diffusion import GaudiStableDiffusionPipeline
```

Comment:
Please do not add ___init for specific pipeline. Follow previously used practice and update `optimum/habana/diffusers/__init__.py`

Reply:
Removed.